mod-search/src/test/resources/spitfire/mod-search/browse/contributor-browse.feature

```diff
@@ -290,6 +290,15 @@ Feature: Tests that browse by contributors
         "name": "Brie",
         "contributorTypeId": ["6e09d47d-95e2-4d8a-831b-f777b8ef6d81"]
       },
+      {
+        "isAnchor": false,
+        "totalRecords": 1,
+        "contributorTypeId": [
+          "6e09d47d-95e2-4d8a-831b-f777b8ef6d81"
+        ],
+        "name": "Brie.",
+        "contributorNameTypeId": "e8b311a6-3b21-43f2-a269-dd9310cb2d0a"
+      },
```

Comment:
There shouldn't be 2 same values in the response. Probably there is some bug on mod-search side. Please check it

Reply:
They are not the same, they have different names:
"Brie."
"Brie"
"brie."